packages/firebase_crashlytics/firebase_crashlytics_platform_interface/pubspec.yaml

```
@@ -8,16 +8,16 @@ environment:
   flutter: ">=1.12.13+hotfix.5 <2.0.0"
 
 dependencies:
+  collection: ^1.14.3
```

This needs bumping to use a nullsafety version

```suggestion
  collection: ^1.15.0-nullsafety.5
```